### PR TITLE
Add Documentation URL for quick access via PyPI web

### PR DESCRIPTION
**Pull request by kerel-fs**
This PR adds a link to the Documentation in the pyproject.toml. This link will be shown in the PyPI web quick links, rendered with a book icon.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -10,6 +10,7 @@ readme.content-type = "text/markdown"
 keywords = ["virtual", "environments", "isolated", "testing"]
 license = "MIT"
 urls.Homepage = "http://tox.readthedocs.org"
+url.Documentation = "https://tox.wiki/"
```
Comment (gaborbernat) [inline pyproject.toml:13]: No need for the trailing /

**Review by gaborbernat — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (jugmac00) [inline pyproject.toml:13]: I am not super familiar with pyproject.toml to configure meta data for projects, but I am pretty sure this should be `urls.Documentation`, right?

See https://hatch.pypa.io/latest/config/metadata/#urls

**Review by jugmac00 — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (kerel-fs) [reply pyproject.toml]: indeed, fixed.

Comment (kerel-fs) [reply pyproject.toml]: how many errors can I put in a one-line change...

thanks! Fixed.

**Review by gaborbernat — APPROVED** [review]

**Pull request merged**